### ISSUE TITLE
`get-urls` is decoding path string, no option to disable

### ISSUE THREAD

**Issue by claytercek** [bluecadet/launchpad#156]
Causes a bug where media items written to disk have different paths than what's written to the JSON

`get-urls` uses `normalize-url` under the hood:

https://github.com/sindresorhus/normalize-url/blob/4869023f4f2766b58e1954fd30e5f1609d2cf8e7/index.js#L184

**Comment by claytercek** [comment]
no longer relevant after #179
